csrankings-i.csv

```diff
@@ -98,7 +98,7 @@ Ido Dagan,Bar-Ilan University,http://u.cs.biu.ac.il/~dagan,YzGAGtoAAAAJ
 Ido Tal,Technion,http://webee.technion.ac.il/people/idotal,PgfGXw4AAAAJ
 Idris Skloul Ibrahim,Heriot-Watt University,https://www.hw.ac.uk/staff/uk/macs/idris-ibrahim.htm,NOSCHOLARPAGE
 Ifat Yasin,University College London,http://www.cs.ucl.ac.uk/cs_people/I.Yasin.html,NOSCHOLARPAGE
-Ifeoma Nwogu,Rochester Institute of Technology,https://www.cs.rit.edu/dr-ifoema-nwogu,pCOmTY0AAAAJ
+Ifeoma Nwogu,University at Buffalo,https://cubs.buffalo.edu/~inwogu/,pCOmTY0AAAAJ

```

Comment:
Research faculty member, not TT, so not eligible for inclusion. Please remove.

Reply:
She is new TT hire from RIT. She was a research faculty member at UB many years ago. Her website is outdated. As shown https://engineering.buffalo.edu/computer-science-engineering/people/faculty-directory.html, she is a TT associate professor.